src/main/java/hillel/spring/doctor/DoctorController.java

```diff
@@ -89,4 +109,10 @@ public void deleteDoctor(@PathVariable Integer id) {
         doctorService.deleteDoctor(id);
     }
 
+    @Bean
+    public RestTemplate resTemplate() {
```

Comment:
Controller is a wrong place for bean declaration. Move it to some `@Configuration` class, or to `Application` one